client/components/common/perfomance-card/EditRatingForm.tsx

```diff
@@ -0,0 +1,141 @@
+import { ApolloError } from 'apollo-client';
+import { useSnackbar } from 'notistack';
+import React, { memo, useState, useEffect } from 'react';
+import { useMutation, useQuery } from 'react-apollo';
+import { Button, TableBody, TableCell, TableRow, Typography } from '@material-ui/core';
+import RatingMUI from '@material-ui/lab/Rating';
+
+import { VOTE_CURRENT_USER } from 'apollo/mutations';
+import { FIND_VOTE_CURRENT_USER } from 'apollo/queries';
+import { useToggle } from 'client/hooks';
+import { FindVoteCurrentUserData, PerformancePost, PollVote, QueryFindVoteCurrentUserArgs, Rating } from 'client/types';
+import { SnackbarErrorText } from 'components/common/snackbar-error-text';
+
+import { AsyncTable } from '../table';
+import { useStyles } from './styles';
+
+
+interface Props {
+    item: PerformancePost;
+}
+
+type RatingState = Rating;
+
+const DEFAULT_RATING: RatingState = {
+    format: 0,
+    content: 0,
+    interest: 0
+};
+
+const EditRatingForm = memo(({ item: { id } }: Props) => {
+    const styles = useStyles();
+    const { enqueueSnackbar } = useSnackbar();
+
+    const [rating, setRating] = useState<RatingState>(DEFAULT_RATING);
+    const { format, content, interest } = rating;
+
+    const [isActivePoll, changeIsActivePoll] = useToggle(true);
```

Comment:
```suggestion
    const [isActivePoll, setIsActivePoll] = useState(true);
```

Reply:
или даже лучше наверное 
```suggestion
    const [isPollActive, setIsPollActive] = useState(true);
```